api/configuration/v1alpha1/kongpluginbinding_types.go

```diff
@@ -22,6 +22,22 @@ import (
 	konnectv1alpha1 "github.com/kong/kubernetes-configuration/api/konnect/v1alpha1"
 )
 
+// KongPluginBindingScope defines the scope of the plugin binding.
+// Allowed values are:
+// - OnlyTargets
+// - GlobalInControlPlane
+// +kubebuilder:validation:Enum=OnlyTargets;GlobalInControlPlane
+type KongPluginBindingScope string
+
+const (
+	// KongPluginBindingScopeOnlyTargets is the scope for the plugin binding to be applied only to the targets.
+	KongPluginBindingScopeOnlyTargets KongPluginBindingScope = "OnlyTargets"
+
+	// KongPluginBindingScopeGlobalInControlPlane is the scope for the plugin binding to be applied to all entities in the
+	// control plane (a.k.a. global scope).
```

Comment:
Written like the above, we have in YAMLs 26 occurrences for `Control Plane` we have 78

Reply:
Why not? 😄  But tbh it probably will unstandardize pretty quickly (unless we enforce it). It's a nit, so it can be left without any harm

Reply:
I'll leave it for now then.